docs/tutorials/uky_flowsheet-solution.ipynb

```diff
@@ -1204,22 +1194,13 @@
       "WARNING (W1002): Setting Var 'fs.leach.liquid[0.0,2].conc_mol_comp[Dy]' to a\n",
```

Comment:
I could skip the validation of the variables. However, I don't know if this would be a positive change. Maybe we should add an issue. What do you think @MarcusHolly

Reply:
We can create an issue in the spirit of getting this PR merged today or tomorrow.